front-end/src/views/Register.vue

```diff
@@ -1,15 +1,108 @@
-<script setup>
-
-
-</script>
-
 <template>
-  <div id="register">
+
+  <div>
     <h1>Register</h1>
-    <router-link to = "/">Home</router-link>
+      <form @submit.prevent="register">
+      <label for="username">Username:</label>
+      <input type="text" id="username" v-model="username" />
+      <br />
+      <label for="email">Email:</label>
+      <input type="email" id="email" v-model="email" />
+      <br />
+      <label for="password">Password:</label>
+      <input type="password" id="password" v-model="password" />
+      <br />
+      <label for="confirmPassword">Confirm Password:</label>
+      <input type="password" id="confirmPassword" v-model="confirmPassword" />
+      <br />
+      <button type="button" @click="register">Register</button>
+    </form>
+    </div>
+    <div class="loginLink">
+    Already have an account? 
+    <router-link to = "/Login">Login here</router-link><br><br>
+    <router-link to = "/" class="home">Home</router-link>
   </div>
 </template>
 
-<style scoped>
+<script>
```

Comment:
looks good to me, but I request you to change the javascript style here in register
Here's the link where you can refer and change the style of javascript accordingly.
https://vuejs.org/guide/extras/composition-api-faq.html

Reply:
As we discussed i didnt write any code for specific registration page but i linked it with login page so in login page i already did javascript style.

Reply:
please refer the above link so that you can make changes to the javascript style

Reply:
Okay i made changes and i will create a new PR.